beetsplug/discogs.py

```diff
@@ -231,7 +231,10 @@ def get_track_from_album(
         return track_info
 
     def item_candidates(
-        self, item: Item, artist: str, title: str
```

Comment:
Nothing seems to have changed in this line. Re your own comment: https://github.com/beetbox/beets/pull/6184#discussion_r2554118022